### PR TITLE
Improve English translation.

### PR DESCRIPTION
**Pull request by t123yh**

### CODE REVIEW

**Comment by gyteng** [comment]
May be you should discuss for the translation, I won't give any suggestion because of my terrible English.

[https://github.com/shadowsocks/shadowsocks-manager/pull/262](https://github.com/shadowsocks/shadowsocks-manager/pull/262)

File: plugins/webgui/public/translate/en-US.js [hunk 1]
```diff
@@ -2,39 +2,39 @@ module.exports = {
   '首页': 'Home',
   '登录': 'Sign in',
   '注册': 'Sign up',
-  '邮箱': 'Email',
+  '邮箱': 'E-mail',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: Not necessary. "Email" is the correct spelling.

**Comment by gyteng** [comment]
实际上对于中文我也有个疑惑的地方：
登陆/登录，账号/帐号，到底应该使用哪个？

**Comment by t123yh** [comment]
login 应该是“登录”。“账号”和“帐号”好像无所谓，不过“帐号”的使用要多一些。

**Comment by t123yh** [comment]
更正：是“账号”使用更多。

File: plugins/webgui/public/translate/en-US.js [hunk 2]
```diff
@@ -66,15 +66,15 @@ module.exports = {
...
+  '最新注册用户': 'Last Registered Users',
+  '最近登录用户': 'Last Logined Users',
   '支付宝订单': 'Alipay Orders',
   'PayPal订单': 'PayPal Orders',
   '订单号：': 'Order No.:',
   '订单类型：': 'Order Type:',
   '金额：': 'Sum:',
-  '用户名：': 'User Name:',
-  '创建时间：': 'Create Time:',
+  '用户名：': 'Username:',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: Either is OK, as long as it is consistent here.

File: plugins/webgui/public/translate/en-US.js [hunk 3]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
-  '请输入正确的用户名格式': 'The right format of user name is required',
-  '密码重试次数已达上限\n请稍后再试': 'The limit of invalid password attempts has been reached\n Please try it later',
-  '验证码发送错误': 'Code sending error',
-  '发送错误，请更换邮箱尝试': 'Code sending error: change your email and try it again',
-  '请求过于频繁，请稍后再试': 'Too frequent requests, please try it later',
-  '当前时段尚未开放注册': 'The registration is closed currently',
-  '重置密码链接已发至您的邮箱，\n请注意查收': 'A password reset link has been sent to your email,\n please check it ',
-  '重置密码链接已经发送，\n请勿重复发送': 'The password reset link has been sent, \n please do not request it again',
-  '请输入正确的邮箱地址': 'A right format of email is required',
+  '网络异常，请稍后再试': 'Network failure. Please try again later.',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: That's OK!

File: plugins/webgui/public/translate/en-US.js [hunk 4]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
+  '用户名或密码错误': 'Incorrect username or password.',
+  '该用户尚未注册的': 'No such user',
+  '请输入正确的用户名格式': 'Username invalid.',
+  '密码重试次数已达上限\n请稍后再试': 'Too many login attempts.\n Please try again later.',
+  '验证码发送错误': 'Unable to send verification code.',
+  '发送错误，请更换邮箱尝试': 'Unable to send verification code. Try to use another E-mail address.',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: Don't forget to use 'please'. Again, 'email address'. 'Please try a different email address'

File: plugins/webgui/public/translate/en-US.js [hunk 5]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
+  '请求过于频繁，请稍后再试': 'Too many requests. Please try again later.',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: That's OK. :-)

File: plugins/webgui/public/translate/en-US.js [hunk 6]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
+  '当前时段尚未开放注册': 'Registration is currently unavailable.',
+  '重置密码链接已发至您的邮箱，\n请注意查收': 'A password reset link has been sent to your E-mail address.\n Please check your inbox, including spam folder.',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: I'd rather use the previous version

File: plugins/webgui/public/translate/en-US.js [hunk 7]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
+  '请输入正确的邮箱地址': 'E-mail address invalid.',
 
   '已过期': 'expired',
 
-  '支付成功': '',
-  '订单会在两分钟内生效，请稍候': '',
-  '系统错误': '',
-  '生成支付订单出错，请稍后再试': '',
+  '支付成功': 'Payment successful.',
+  '订单会在两分钟内生效，请稍候': 'The order is taking effect in 2 minutes. Please be patient.',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: 'Please wait'.

File: plugins/webgui/public/translate/en-US.js [hunk 8]
```diff
@@ -88,23 +88,23 @@ module.exports = {
...
+  '系统错误': 'System error',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: A system error occurred.

File: plugins/webgui/public/translate/en-US.js [hunk 9]
```diff
@@ -2,39 +2,39 @@ module.exports = {
...
-  '易于配置':'Config easily',
+  '快速搭建':'Fast deployment',
+  '易于配置':'Easy configuration',
   '官方标准':'Official API',
-  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Just depend on Node.js and SQLite (MySQL optionally)',
-  '带有插件系统，仅需修改配置文件即可运行': 'Based on plugins, just run it after editing the config file',
-  '支持libev和python版本的标准manager API': 'Support the standard manager API of both ss-libev and ss-python',
-  '请输入邮箱地址再点击“找回密码”': 'Please fill in your email address first',
+  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Depends on Node.js and SQLite (optionally MySQL)',
+  '带有插件系统，仅需修改配置文件即可运行': 'Plugin-based, super-easy deployment',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: Could you please be more specific about the problem?

File: plugins/webgui/public/translate/en-US.js [hunk 10]
```diff
@@ -2,39 +2,39 @@ module.exports = {
...
+  '支持libev和python版本的标准manager API': 'Support standard manager API of both ss-libev and ss-python',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: Yeah, sure.

File: plugins/webgui/public/translate/en-US.js [hunk 11]
```diff
@@ -2,39 +2,39 @@ module.exports = {
...
+  '请输入邮箱地址再点击“找回密码”': 'Please fill in your E-mail address first',
   '邮箱不能为空': 'Email address cannot be empty',
   '验证码不能为空': 'Code cannot be empty',
   '密码不能为空': 'Password cannot be empty',
-  '验证码已发至邮箱': 'A verification code has been sent to your email',
-  '用户注册成功':'Your registration is done',
+  '验证码已发至邮箱': 'A verification code has been sent to your E-mail account',
+  '用户注册成功':'Successfully registered.',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: I have always seen 'sent to your Email account' or 'address'.

File: plugins/webgui/public/translate/en-US.js [hunk 12]
```diff
@@ -2,39 +2,39 @@ module.exports = {
...
   '确定': 'OK',
   '取消': 'Cancel',
   '关闭': 'Close',
   '下一步': 'Next',
 
-  '服务器': 'Server',
-  '用户': 'User',
-  '账号': 'Account',
-  '订单': 'Order',
+  '服务器': 'Servers',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: No they are not. They are hyperlinks to their pages.

**Comment by CicoZhang** [comment]
@gyteng  Good to see these screenshots. Plural forms should be used.

File: plugins/webgui/public/translate/en-US.js [hunk 13]
```diff
@@ -66,15 +66,15 @@ module.exports = {
   '手机请点击二维码付款': 'On your smartphone, please click the QR code to pay',
   'Paypal支付请点击下面按钮': 'For Paypal users, please click the following button',
 
-  '最新注册用户': 'Last Registered User',
-  '最近登录用户': 'Last Logon User',
+  '最新注册用户': 'Last Registered Users',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: They are plurals. I'll post a screenshot later.

File: plugins/webgui/public/translate/en-US.js (hunk 2 above)
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: See https://english.stackexchange.com/questions/43436/username-user-name-or-user-name

File: plugins/webgui/public/translate/en-US.js [hunk 14]
```diff
@@ -44,18 +44,18 @@ module.exports = {
   '流量：': 'Data Usage:',
   '周期：': 'Duration:',
   '到期时间：': 'Due Date:',
-  '最近连接：': 'Recent Connection:',
+  '最近连接：': 'Last Connected:',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: In Telegram we see `Last seen on xxx`, so I assume `Last Connected` should be fine.

**Comment by gyteng** [comment]
订单状态共有5种，当然翻译成英文后肯定无法这么整齐。

File: plugins/webgui/public/translate/en-US.js (hunk 11 above)
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: @t123yh I have seen them too. English is flexible, either way is OK, as long as it consistent. No point waste time on the 'personal taste'.

File: plugins/webgui/public/translate/en-US.js (hunk 14 above)
Comment (gyteng) [inline plugins/webgui/public/translate/en-US.js]: It means the last time he used the server.

Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: @t123yh  The problem is which is the subject?  The case does not seem to be transferable here.

File: plugins/webgui/public/translate/en-US.js (hunk 2 above)
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: Believe or not, either is OK as long as it is consistent.

**Comment by CicoZhang** [comment]
@t123yh @gyteng  Let's stick to 'email address' consistently, shall we?

File: plugins/webgui/public/translate/en-US.js (hunk 12 above)
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: @t123yh I've seen the screenshots posted below by @gyteng , I go for plural forms now.

File: plugins/webgui/public/translate/en-US.js [hunk 15]
```diff
@@ -44,18 +44,18 @@ module.exports = {
...
   '或': 'or',
   '点击这里': 'Click Here',
-  '付款立即开通帐号': 'pay and assign an account now',
-  '点击二维码或者用移动设备扫描二维码可自动填充服务器信息': 'Click the QR code or scan it by smartphone to automate the retrieval of server information',
-  '目前该用户没有分配账号，请联系管理员处理': 'The current user has not been assigned an account, please contact the administrator',
-  '请选择续费周期：': 'Please choose the length of service:',
+  '付款立即开通帐号': 'pay and get an account now',
+  '点击二维码或者用移动设备扫描二维码可自动填充服务器信息': 'Click the QR code or scan it by smartphone to retrieve server information automatically',
+  '目前该用户没有分配账号，请联系管理员处理': 'No account available yet. Please contact the administrator.',
+  '请选择续费周期：': 'Service duration:',
```
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: Yeah, I think this one is worth improving. But the original one doesn't seem to be nice either.

File: plugins/webgui/public/translate/en-US.js (hunk 14 above)
Comment (t123yh) [inline plugins/webgui/public/translate/en-US.js]: In Telegram we mean the user `was last seen`, and here we mean the account `was last connected`.

File: plugins/webgui/public/translate/en-US.js (hunk 9 above)
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: Sorry, I mean Line 15, grammar problem.

**Comment by t123yh** [comment]
@CicoZhang I've pushed some modifications. How about them now?

File: plugins/webgui/public/translate/en-US.js [hunk 16]
```diff
@@ -11,14 +11,14 @@ module.exports = {
   '快速搭建':'Fast deployment',
   '易于配置':'Easy configuration',
   '官方标准':'Official API',
-  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Depends on Node.js and SQLite (optionally MySQL)',
+  '仅依赖Node.js，无需安装数据库（可选MySQL）': 'Depends on Node.js and SQLite only (optionally MySQL)',
   '带有插件系统，仅需修改配置文件即可运行': 'Plugin-based, super-easy deployment',
-  '支持libev和python版本的标准manager API': 'Support standard manager API of both ss-libev and ss-python',
-  '请输入邮箱地址再点击“找回密码”': 'Please fill in your E-mail address first',
+  '支持libev和python版本的标准manager API': 'Supports standard manager API of both ss-libev and ss-python',
+  '请输入邮箱地址再点击“找回密码”': 'Please fill in your Email address first',
```
Comment (CicoZhang) [inline plugins/webgui/public/translate/en-US.js]: 'email', no need capitalizing it.

**Comment by t123yh** [comment]
@CicoZhang all the issues you mentioned are resolved.

**Pull request merged**